draft-ietf-webpush-protocol.xml

```diff
@@ -1084,7 +1084,7 @@ HEADERS      [stream 82] +END_STREAM
             A push service MAY expire a subscription set at any time which MUST also
             expire all push message subscriptions in the set. If a user agent has an
             outstanding request to a push subscription set (<xref target="monitor-set"/>)
-            this can be signaled by returning a 400-series status code, such as 410 (Gone).
+            this can be signaled by returning a 410 (Gone) status code.

```

Comment:
This should be a 404.
As I responded to JR:

7.3.1 should be 404 and not 410.
410 (Gone) is returned when a push service decides to cease delivery attempts before the TTL expires.
404 (Not Found) is returned when a push services expires a subscription or set.